docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -292,16 +292,8 @@ Used to lookup fungible assets balances
 #### Query parameters
 
 * `projectId` - the project identifier.
-* `currency` - currency to calculate the assets value. e.g. `usd`. Supported currencies:
-    * `btc`
-    * `eth`
-    * `usd`
-    * `eur`
-    * `gbp`
-    * `aud`
-    * `cad`
-    * `inr`
-    * `jpy`
+* `currency` - currency to calculate the assets value. e.g. `usd`.
+    * Supported currencies: `btc`, `eth`, `usd`, `eur`, `gbp`, `aud`, `cad`, `inr`, `jpy`.
```

Comment:
Make this list horizontal for better readability.

Reply:
Is it not horizontal already?